hw/xbox/nv2a/nv2a_user.c

```diff
@@ -59,21 +61,23 @@ uint64_t user_read(void *opaque, hwaddr addr, unsigned int size)
 void user_write(void *opaque, hwaddr addr, uint64_t val, unsigned int size)
 {
     NV2AState *d = (NV2AState *)opaque;
+    unsigned int channel_id;
+    ChannelControl *control;
+    uint32_t channel_modes;
```

Comment:
Same as above: Prefer declaration and initialization together for this function.